README.md

```diff
@@ -1,2 +1,6 @@
 # city-sr-api-example
```

Comment:
A good README file should tell someone looking at the code repo for the first time:

1. What is the intention of the code repo
2. Provide some guidance as to how the repo is laid out
3. Provide instructions for how to install and make use of the code in the repo.

Please add content along these lines